libs/natparse/src/main/java/org/amshove/natparse/natural/IGetTransactionNode.java

```diff
@@ -0,0 +1,8 @@
+package org.amshove.natparse.natural;
+
+import org.amshove.natparse.ReadOnlyList;
+
+public interface IGetTransactionNode extends IStatementNode
```

Comment:
this needs to `extends IStatementNode, IMutateVariables`, then you can test it :)

Reply:
of course, thanks